README.md

```diff
@@ -50,7 +50,7 @@ There's a multitude of settings you can use to control the plugin server. Use th
 | KAFKA_CLIENT_CERT_B64         | Kafka certificate in Base64                                | `null`                                |
 | KAFKA_CLIENT_CERT_KEY_B64     | Kafka certificate key in Base64                            | `null`                                |
 | KAFKA_TRUSTED_CERT_B64        | Kafka trusted CA in Base64                                 | `null`                                |
-| KAFKA_PRODUCER_MAX_QUEUE_SIZE | Kafka producer queue max size before flushing              | `1000`                                |
+| KAFKA_PRODUCER_MAX_QUEUE_SIZE | Kafka producer queue max size before flushing              | `20`                                  |
```

Comment:
20 seems like a pretty small number. 🤔 Certainly relative to 1000. But I don't know what the actual sizes tend to be in production. Could we get some metric reporting into DB#flushKafkaMessages?

Reply:
Good idea. 

My plan is to experiment a bit here - hard to judge w/o knowing more about the behavior in production. We might even need a different tactic all-together, e.g. measuring the size of each message individually or something like that.

20 is the lowest we can go - if the error goes away after that we'll have more information.

As such, I'll add the queue size metrics once I have some measurements from this. :)

Reply:
I believe we can't increase max request size, as we're managed by Heroku.
Why not sprinkle in some StatsD now?

Reply:
> I believe we can't increase max request size, as we're managed by Heroku.

Yes, but not for ever. :)

> Why not sprinkle in some StatsD now?

Keep PR lean, don't want to wait too many hours to get to see the result of this.